src/main/resources/templates/fragments/navbar.html

```diff
@@ -0,0 +1,85 @@
+<div th:fragment="navbar" id="navbar">
+    <nav class="navbar">
+        <div class="navbar-brand">
+            <a class="navbar-item" href="/">OurShop</a>
+        </div>
+
+        <!-- Navbar Links -->
+        <div class="navbar-links">
+            <a class="navbar-item" href="/">Home</a>
```

Comment:
You could add the role attribute on each the `<a>` tags for accessibility

Reply:
i looked this up, and it seems that <a> tags have a role='link' per default... (?)

Reply:
in this case they are not only just links but they have a role="menuitem"